pint/models/pint_pulsar_binary.py

```diff
@@ -120,3 +144,10 @@ def binary_delay(self,toas):
         for bdf in self.binary_delay_funcs:
             bdelay+= bdf(toas)
         return bdelay
+
+    @Cache.use_cache
+    def d_delay_d_xxxx(self,param,toas):
+        """Return the DD timing model delay derivtives"""
```

Comment:
The comment to return the 'DD' binary model seems slightly wrong. DD is only the 'default' now, right?